frontend/src/Editor/Container.jsx

```diff
@@ -563,9 +558,8 @@ export const Container = ({
                 currentPageId,
                 childComponents,
                 isVersionReleased,
-                setReleasedVersionPopupState,
+                enableReleasedVersionPopupState,
```

Comment:
It can be removed here. These props are being passed to the `SubContainer` but we're fetching it from the store already in `SubContainer`